### ISSUE TITLE
Question: All encompassing list of differences in production environment. 

### ISSUE THREAD

**Issue by shamasis** [balderdashy/sails#3378]
Hi, 

I understand that I can analyse the code by searching `=== 'production'`. But, if you have this list handy, it would be great if shared. 

I have faced some subtle differences in how logs, validations and debugging features are handled separately in production environment. A list would help from avoiding those "is this a feature, a bug or am I going nuts" moments. :smiley:

**Comment by elssar** [comment]
+1

**Comment by mikermcneil** [comment]
@shamasis awesome idea!  Would you mind sending a PR to the ROADMAP.md file and closing this issue?

**Comment by shamasis** [comment]
Sure. On its way.

**Comment by sailsbot** [comment]
Thanks for posting, @shamasis.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
